tests/setup_test.go

```diff
@@ -1935,3 +1936,68 @@ def main(ctx):
 		}
 	}
 }
+
+func TestUserOrgs(t *testing.T) {
+	dir, err := ioutil.TempDir("", "agola")
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+	defer os.RemoveAll(dir)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	tetcd, tgitea, c := setup(ctx, t, dir)
+	defer shutdownGitea(tgitea)
+	defer shutdownEtcd(tetcd)
+
+	gwClient := gwclient.NewClient(c.Gateway.APIExposedURL, "admintoken")
+
+	_, _, err = gwClient.CreateOrg(ctx, &gwapitypes.CreateOrgRequest{Name: "org01", Visibility: gwapitypes.VisibilityPublic})
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+	_, _, err = gwClient.CreateOrg(ctx, &gwapitypes.CreateOrgRequest{Name: "org02", Visibility: gwapitypes.VisibilityPrivate})
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+	_, _, err = gwClient.CreateOrg(ctx, &gwapitypes.CreateOrgRequest{Name: "org03", Visibility: gwapitypes.VisibilityPublic})
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+
+	_, token := createLinkedAccount(ctx, t, tgitea, c)
+
+	_, _, err = gwClient.AddOrgMember(ctx, "org01", giteaUser01, gwapitypes.MemberRoleMember)
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+	_, _, err = gwClient.AddOrgMember(ctx, "org02", giteaUser01, gwapitypes.MemberRoleOwner)
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+
+	gwClientNew := gwclient.NewClient(c.Gateway.APIExposedURL, token)
+
+	orgs, _, err := gwClientNew.GetUserOrgs(ctx)
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+	if len(orgs) != 2 {
+		t.Fatalf("expected len 2 got: %d", len(orgs))
+	}
+
+	orgsWanted := make([]*gwapitypes.UserOrgsResponse, 0)
+	orgsWanted = append(orgsWanted, &gwapitypes.UserOrgsResponse{
```

Comment:
You can avoid the appends and just create an unique object.

Reply:
done !